core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1178,6 +1178,16 @@ package object config {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(100 * 1024 * 1024)
 
+  private[spark] val SHUFFLE_ACCURATE_SKEWED_BLOCK_THRESHOLD =
+    ConfigBuilder("spark.shuffle.accurateSkewedBlockThreshold")
+      .doc("Threshold in bytes above which the size and five times non-empty average size, " +
+        "the skewed shuffle blocks in HighlyCompressedMapStatus is accurately recorded. " +
+        "This helps to prevent OOM by avoiding underestimating skewed shuffle block size " +
+        "when fetch shuffle blocks.")
+      .version("3.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefault(350 * 1024)
```

Comment:
Could you describe the background of this value?

Reply:
> Could you describe the background of this value?

Thanks for review. Described in the reply to @mridulm below, hopes to get your opinion.